docker-compose.yml

```
@@ -19,5 +19,8 @@ denguetorpedo:
   container_name: denguetorpedo
   env_file: .env
   volumes:
-  - <local_folder_of_cloned_denguetorpedo_git_project>:/home/dengue/denguetorpedo
-  restart: always
\ No newline at end of file
+  - /home/fernando/Documentos/SE2/denguetorpedo:/home/dengue/denguetorpedo
```

Do not include local paths or info. Keep it general and just indicate what has to be changed/replaced, and how.


```suggestion
  - <local_folder_of_cloned_denguetorpedo_git_project>:/home/dengue/denguetorpedo
```